docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,4 +1,111 @@
 [[logs-troubleshooting]]
 = Troubleshoot your logs
 
-_coming soon_
\ No newline at end of file
+This section provides solutions to errors you might encounter while onboarding your logs in {kib}.
+
+[discrete]
+[[logs-troubleshooting-insufficient-priv]]
+== User does not have permissions to create API key
+
+If you don't have the required privileges to create an API key, you'll see the following error message:
+
+----
```

Comment:
All of these code blocks need `[source,*]` to render correctly. Maybe `[source,plaintext]`? A full list of options is available [here](https://github.com/rouge-ruby/rouge/blob/master/docs/Languages.md) if you want to look through them.